boards/x86_64/intel64/qemu-intel64/configs/ostest/defconfig

```diff
@@ -1,114 +1,53 @@
 #
-# Automatically generated file; DO NOT EDIT.
-# Nuttx/x86_64 Configuration
+# This file is autogenerated: PLEASE DO NOT EDIT IT.
 #
```

Comment:
A lot of changes to this defconfig.  This means that it was probably hand editted.  Running refresh.sh on hand-edited defconfig files usually breaks them because a human is not capable of keeping all of the dependencies in their head.